pkg/diff/schema_migration_plan_test.go

```diff
@@ -838,6 +838,48 @@ var (
 			expectedStatements: nil,
 			expectedDiffErrIs:  errDuplicateIdentifier,
 		},
+		{
+			name: "Online check constraint addition",
```

Comment:
thought: Do we already have a case for the validation of a previously invalid constraint?

Reply:
This is a different test case: 
- one is testing altering a check constraint from invalid to valid
- one is testing that add a constraint is online